### PR TITLE
Solved the task

### PR DESCRIPTION
**Pull request by Cerne13**
Driver list w/search
image

Car list w/search
image

Manufacturer list w/search
image

### CODE REVIEW

File: taxi/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer
+
```
Comment (Gr1zzZzzly) [inline taxi/tests/test_views.py:6]: add tests for search plz)

**Review by Gr1zzZzzly — APPROVED** [review]

File: taxi/tests/test_views.py (hunk 1 above)
Comment (Gr1zzZzzly) [reply taxi/tests/test_views.py:6]: sory wrong approve))

**Review by Gr1zzZzzly — CHANGES_REQUESTED** [review]

File: taxi/tests/test_views.py (hunk 1 above)
Comment (Cerne13) [reply taxi/tests/test_views.py:6]: 👍

**Review by MasterpieceElbow — APPROVED** [review]
GJ!

**Pull request closed without merging**